includes/Plugin.php

```diff
@@ -158,6 +160,7 @@ function() use ( $options, $transients, $assets ) {
 				( new Core\Notifications\Notifications( $this->context, $options, $authentication ) )->register();
 				( new Core\Util\Debug_Data( $this->context, $options, $user_options, $authentication, $modules ) )->register();
 				( new Core\Admin\Standalone( $this->context ) )->register();
+				( new Core\Util\Activation( $this->context, $options, $assets ) )->register();
```

Comment:
This is too late for `Activation` to hook on to `googlesitekit_activation`, which is causing the activation notification not to show up.

Reply:
I split the class into two so that the one class can take care of activation logic while the other one handles the notice. This also makes sense because only the latter requires the `Assets` instance, hence should be instantiated later.